src/components/DeviceProgram.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import Time from "./Time";
+import Volume from "./Volume";
+import Tidal from "./Tidal";
+import Settings from "./Settings";
+
+const DeviceProgram = () => {
+  const [activeComponent, setActiveComponent] = useState("Time");
+
+  const renderComponent = () => {
+    switch (activeComponent) {
+      case "Time":
+        return <Time />;
+      case "Volume":
+        return <Volume />;
+      case "Tidal":
+        return <Tidal />;
+      case "Settings":
+        return <Settings />;
+      default:
+        return <Time />;
+    }
+  };
+
+  return (
+    <div className="p-4">
+      <div className="flex space-x-2 mb-0 border border-gray">
+        {["Time", "Volume", "Tidal", "Settings"].map((item) => (
```

Comment:
can read this array of strings from a const file

Reply:
done